home/rvagg/2021-02-exploration-report-small-filecoin-deals.md

```diff
@@ -0,0 +1,134 @@
+# @rvagg Exploration Report: Small Filecoin Deals (2021-02)
+
+## Background
+
+### Filecoin storage for the uninitiated
+
+* Typical miners are continually "sealing" new sectors, containing either 32 GiB or 64 GiB of data
+* Sealed sectors have a lifetime, pre-specified by the miner, up to a maximum of 18 months and a minimum of 6 months
+* Sealed sectors contribute to miner "power", power increases a miner's chances of winning block rewards
+* Sectors may, or may not contain data from clients in the form of paid "deals"
+* Sectors not containing deals are called "committed capacity" (CC) sectors, they simply contribute to the total network capacity without storing useful data
```

Comment:
> This is a reason why miners prefer 16.1G+ fil+ deals - it leaves no CC space at all

from what I understand, deals _must_ be ^2 sized, so this would be 32G wouldn't it? So the prefernce here would simply come from the fact that they are able to fill up a sector with a deal while only transferring the minimum amount of data from the client? is the data transfer the only benefit here?

Reply:
>  the fact that they are able to fill up a sector with a deal while only transferring the minimum amount of data from the client? 

given how difficult it is currently - that's a *very* big deal ;)